### PR TITLE
Implementation cloudinary

### PR DESCRIPTION
**Pull request by Joe-woods**
voici mon form pour les new nvt

### CODE REVIEW

**Review by MBOZAR — APPROVED** [review]

**Review by TheRealCL — APPROVED** [review]
all good

**Comment by Joe-woods** [comment]
update cloudinary

File: config/environments/development.rb [hunk 1]
```diff
@@ -8,6 +8,9 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
+  # cloudinary
+  config.active_storage.service = :cloudinary
```
Comment (Rox1610) [inline config/environments/development.rb:12]: same as to be done here:

# config/environments/production.rb
# [...]
config.active_storage.service = :cloudinary

**Review by Rox1610 — CHANGES_REQUESTED** [review]
one action to be done

**Review by MBOZAR — CHANGES_REQUESTED** [review]
re-push production.rb

**Review by Rox1610 — APPROVED** [review]
ok good to go

**Review by MBOZAR — APPROVED** [review]
yessir

**Pull request merged**